### DISCUSSION TITLE
Scythe Anticheat v2.20.1

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#125] [Announcements]
Scythe Anticheat is now at version v2.20.1!

Supported Minecraft Versions: 1.20.60

This bug fix is only necessary if your realm/server enabled BadEnchantsD, which is disabled by default.

# Changelog
- Fix a critical issue with BadEnchantsD that would incorrectly flag items. 

By downloading this addon, you agree to our [license](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/blob/main/LICENSE)
Questions? Join our [discord](https://discord.gg/pKyu95mYCT)!

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/v2.20.1'>Scythe Anticheat v2.20.1</a>.</em>

**Comment by WaddleDeveloper7** [comment]
Hey would you mind updating it so that command block expoits are turned off by default? My friends keep getting flagged for it even though Mojang patched it. And for some reason when I set it to "false" it still doesn't work. Thanks so much, this anticheat is super helpful for my realm! :)
